kafka-commons/src/test/java/uk/gov/homeoffice/digital/sas/transactionsync/KafkaDbTransactionSyncronizerIntergrationTest.java

```diff
@@ -0,0 +1,141 @@
+package uk.gov.homeoffice.digital.sas.transactionsync;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.homeoffice.digital.sas.kafka.constants.Constants.DATABASE_TRANSACTION_SUCCESSFUL;
+import static uk.gov.homeoffice.digital.sas.kafka.constants.Constants.KAFKA_TRANSACTION_INITIALIZED;
+import static uk.gov.homeoffice.digital.sas.kafka.constants.Constants.TRANSACTION_SUCCESSFUL;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
+import java.util.Random;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.kafka.test.context.EmbeddedKafka;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.transaction.support.TransactionSynchronizationManager;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+import uk.gov.homeoffice.digital.sas.config.TestConfig;
+import uk.gov.homeoffice.digital.sas.kafka.transactionsync.KafkaDbTransactionSynchronizer;
+import uk.gov.homeoffice.digital.sas.model.Profile;
+import uk.gov.homeoffice.digital.sas.repository.ProfileRepository;
+
+@SpringBootTest(classes = TestConfig.class)
+@DirtiesContext
+@EmbeddedKafka(
+    partitions = 1,
+    brokerProperties = {
+        "listeners=PLAINTEXT://localhost:3333",
+        "port=3333"
+    }
+)
+class KafkaDbTransactionSyncronizerIntergrationTest {
```

Comment:
typo: missing 'h' in Sync**h**ronizer

Reply:
Actioned

Reply:
ah it looks like you've accidentally placed the 'h' after the 'r'.